saleor/graphql/webhook/subscription_types.py

```diff
@@ -953,6 +954,52 @@ class Meta:
         interfaces = (Event,)
 
 
+class PromotionBase(AbstractType):
+    promotion = graphene.Field(
+        "saleor.graphql.discount.types.Promotion",
+        description="The promotion the event relates to.",
+    )
+
+    @staticmethod
+    def resolve_promotion(root, info: ResolveInfo, channel=None):
+        _, promotion = root
+        return promotion
+
+
+class PromotionCreated(SubscriptionObjectType, PromotionBase):
+    class Meta:
+        root_type = "Promotion"
+        enable_dry_run = True
+        interfaces = (Event,)
+        description = "Event sent when new promotion is created." + ADDED_IN_315
```

Comment:
Shouldn't we mark these events as a preview_feature also? All of them are called by the mutations that are `preview_feature`.